### DISCUSSION TITLE
Multi view 3D Reconstruction

### DISCUSSION THREAD

**Discussion by AmrElsersy** [facebookresearch/pytorch3d#1512] [Q&A]
Hi, I have a video of capturing human body in 360 degree (rotated camera around the person)

I want to generate a 3d model from this multi view images of the person, how can I do that ??

I tried the voxel carving method and SFM method but it seems it gives bad results, is there any deep learning based methods for that ?

**Comment by HarryWei1998** [comment]
You can usage the hot method named NeRF. Good luck for you.
